src/BetterAppListBuilder.php

```diff
@@ -1,10 +1,11 @@
 <?php
 
+declare(strict_types = 1);
+
+namespace Drupal\apigee_edge_ui;
+
 /**
- * @file
- * Apigee Edge UI module for Drupal.
- *
- * Copyright (C) 2019 PRONOVIX GROUP BVBA.
+ * Copyright (C) 2020 PRONOVIX GROUP BVBA.
```

Comment:
If a file was added to the repo in 2019 then the copyright header should remain 2019, only new files should have 2020.